### PR TITLE
📝  Added documentation for workstacks

### PR DESCRIPTION
**Pull request by robertdeniszczyc2**

### CODE REVIEW

File: documentation/workstacks.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+[[toc]]
+
+## What is a workstack?
+A workstack displays a collection of cases to the user.
+
+## What workstacks are available?
+There are several types of workstacks available.
+
+### Team
+Displays all the open cases allocated to a specific team. Team workstacks for the teams the user is assigned to are displayed to the user on the main Dashboard.
```
Comment (LiamMacP) [inline documentation/workstacks.md:17]:
```suggestion
Displays all the open cases allocated to a specific team. User-assigned team workstacks are displayed to the user on the main Dashboard.
```

File: documentation/workstacks.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Workstack configuration
+Workstacks are configured in the database schema and are made up of [types](https://github.com/UKHomeOffice/hocs-info-service/blob/main/src/main/java/uk/gov/digital/ho/hocs/info/domain/model/WorkstackType.java) and [columns](https://github.com/UKHomeOffice/hocs-info-service/blob/main/src/main/java/uk/gov/digital/ho/hocs/info/domain/model/WorkstackColumn.java).
+
+Column configurations are defined at team level, casetype level, user workstacks for a specific case type, and search results.
+
+If a workstack is loaded with no defined configuration, the system will fall back to using a system-wide default configuration.
+
+### Search workstack type
+For Search workstacks, the type must be defined with prefixed concatenated case types in alphabetical order of case type, for example `ALPHA,BRAVO_SEARCH_RESULTS`.
```
Comment (LiamMacP) [inline documentation/workstacks.md:42]: Are these alphabetical?

**Review by LiamMacP — APPROVED** [review]
Looks good, one question and minor alteration (if agreed)

File: documentation/workstacks.md (hunk 2 above)
Comment (robertdeniszczyc2) [reply documentation/workstacks.md:42]: How about like instead:
```suggestion
For Search workstacks, the type must be defined with prefixed concatenated case types in alphabetical order of case type, for example `A,B_SEARCH_RESULTS`.
```

Comment (LiamMacP) [reply documentation/workstacks.md:42]: I was happy with the wording - didn't know for example if we did `B,A_SEARCH_RESULTS` anywhere. I've just checked and we don't - so happy to resolve.

**Pull request merged**